src/components/RadioButton/RadioButton.vue

```diff
@@ -1,48 +1,52 @@
 <template>
-  <div class="inline-block mr-6 mt-1">
-    <input
-      :id="id"
-      type="radio"
-      :class="[
-        'm-0 p-0 opacity-0 mt-2',
-        {'radio__input--error': error},
-        {'!cursor-not-allowed': disabled}
-      ]"
-      :name="name"
-      :value="value.toString()"
-      :checked="checked"
-      :disabled="disabled"
-      :required="required"
-      @input="onInput"
-      @click="onClick"
+  <div
+    :class="[
+      'w-fit focus-within:outline-dashed focus-within:outline-1 py-1 mr-5 h-[28px]',
+      {'!h-[46px]' : helperText}
+    ]"
+  >
+    <div
```

Comment:
trying to understand why it needs two div wrappers?

Reply:
If I add bg-color to this div I see this odd absoluted shape and wondering if we can avoid it
<img width="145" alt="Screen Shot 2022-11-28 at 4 18 33 PM" src="https://user-images.githubusercontent.com/50080618/204383039-bf7e6834-24a0-4d69-945e-73c3c4ed9b89.png">

Reply:
also wondering if the outer div should still be inline-block although I can't remember why it is there in the first place. doesn't seem to add any value here

Reply:
try removing this div and add a pr-10 to the label instead?

Reply:
> If I add bg-color to this div I see this odd absoluted shape and wondering if we can avoid it

That would be impossible to avoid - what's going on is this is relatively positioned because there is the native radio button in the background which has to have 0 opacity

Reply:
try removing <div class="inline-block mr-12"> completely and add pr-10 to the label - this seemed to work - unless I am missing somehting more